rest/views.py

```diff
@@ -48,6 +49,39 @@
 from django.http.response import Http404
 
 
+def get_urls(raw_urls, list_urls, urlbase=''):
+    """
+    Recursively builds a list of all the urls in the current project and
+    the name of their associated view
+    """
+    list_urls = list_urls or []
```

Comment:
I guess here you mean that `list_urls` can be `None`.
If this is the case you should change the definition to `def get_urls(raw_urls, list_urls=None, urlbase='')`  otherwise this line seems not useful.